ArduSub/mode.cpp

```diff
@@ -155,6 +155,7 @@ bool Sub::set_mode(const uint8_t new_mode, const ModeReason reason)
 // called at 100hz or more
 void Sub::update_flight_mode()
 {
+    pos_control.update_estimates();
```

Comment:
maybe we should do the same here as we've done for plane which is update this at the same time as inertial_nav.update() is called.  Same for blimp.

Reply:
I have added this as a seperate commit so both options are there for discussion.